projects/plugins/jetpack/class.jetpack.php

```diff
@@ -846,6 +846,7 @@ public function configure() {
 
 		$config->ensure( 'search' );
 		$config->ensure( 'wordads' );
+		$config->ensure( 'publicize' );
```

Comment:
I think we need to add [the config method](https://github.com/Automattic/jetpack/tree/master/projects/packages/config#the-feature-enabling-method) to the Config package, and the configure method to Publicize. I'm not sure what we'll want to do in there, but it could do things like set up the `$publicize` global or instantiate the UI class (as I think the global gets created in there).